eclipse-collections/src/main/java/org/eclipse/collections/impl/list/Interval.java

```diff
@@ -176,6 +176,26 @@ public static Interval fromTo(int from, int to)
         return Interval.fromToBy(from, to, -1);
     }
 
+    /**
+     * Returns an Interval starting from the value from until the specified value to (exclusive) with a step value of 1
+     */
+    public static Interval fromUntilTo(int from, int to)
```

Comment:
should the name be `fromToExclusive()`?

Reply:
I thought you were okay with some other signature for the API. Let me use `fromToExclusive`

Reply:
I am fine with the naming. Naming is hard, so I’ll let others also opine on naming

Reply:
I like `fromToExclusive` better.

Reply:
Took the `until` inspiration from the Kotlin. Changed the API to `fromToExclusive` to align with other APIs. :)